### DISCUSSION TITLE
PHP mail() function

### DISCUSSION THREAD

**Discussion by mis-uep** [phalcon/cphalcon#15396] [Q&A]
Hello, how can we use PHP mail() function in PHALCON?

**Comment by quasipickle** [comment]
The mail() function is extremely rudimentary and generally should not be used in an application.  I strongly recommend PHPMailer or Swiftmailer.

**Reply by mis-uep** [reply]
Thanks!

**Reply by mis-uep** [reply]
any idea how to fix 500 internal server error in VPS? I use Swiftmailer.

**Reply by quasipickle** [reply]
A 500 error can be anything.  You should check you server logs to get some insight.

**Reply by mis-uep** [reply]
Another question. Im using swiftmailer before asking the question above. My problem at first is I cant send email using our new server, we've migrated from GCP to VPS of hostgator.

Now, I switched from using Swift_SmtpTransport to Swift_SendmailTransport and it worked! but the problem is, the email is going directly to SPAM folder of the recipient.  Any fix for this? 

Thanks!

**Reply by BeMySlaveDarlin** [reply]
SPAM issue isn't related to sendmail or mail services directrly.
There is quite old and complete guides over so and quora, showing how to avoid your mails marked as spam.
